src/core/lib/iomgr/error_internal.h

```diff
@@ -22,7 +22,7 @@
 #include <grpc/support/port_platform.h>
 
 #include <inttypes.h>
-#include <stdbool.h>  // TODO, do we need this?
+#include <stdbool.h>  // TODO(unknown): , do we need this?
```

Comment:
Remove unnecessary comma.

Also, consider assigning this TODO to yourself, as the owner of the grpc_error code. :)

Reply:
Done